From 910ee392cdb8480bcc905c31b2900f257edb88da Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Tue, 13 Oct 2015 15:58:55 +0800
Subject: [PATCH 0563/5242] MLK-11702-1 ARM: dts: imx6sx-sabreauto: add dts
 support for pxp

commit  bec28fe8f19bea40de434779ba88b63d504ea0a0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pxp module on the imx6sx sabreauto
board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 35d2554..1986e79 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -430,6 +430,10 @@
 	status = "okay";
 };
 
+&pxp {
+	status = "okay";
+};
+
 &iomuxc {
 	pinctrl_egalax_int: egalax-intgrp {
 		fsl,pins = <
-- 
1.7.9.5

